src/services/template/EventResolvers.ts

```diff
@@ -0,0 +1,95 @@
+import { Arg, Ctx, Field, Int, Mutation, ObjectType, Query, Resolver, UseMiddleware } from "type-graphql";
+import { hash, compare } from "bcryptjs";
+import { User } from './entity/User';
+import { Event } from './entity/Event';
+import { createAccessToken, createRefreshToken } from './auth';
+import { MyContext } from './MyContext';
+import { isAuth } from './isAuth';
+import { sendRefreshToken } from './sendRefreshToken';
+import { getConnection } from 'typeorm';
+import { verify } from 'jsonwebtoken';
+
+@ObjectType()
+class NewEvent {
+    @Field()
+    accessToken: string;
+    @Field(()=> Event)
+    event: Event;
+}
+
+@Resolver()
+export class EventResolver {
+    @Query(()=>String)
+    One() { 
+        return "hello";
+    }
+    
+    @Query(()=>String)
+    @UseMiddleware(isAuth)
+    bye(
+        @Ctx() {payload}: MyContext
+    ) {
+        return `bye, ${payload.userId}`;
+    }
+
+    @Query(()=> [Event])
+    events() {
+        return Event.find();
+    }
+    
+    @Mutation(() => Boolean)
+    async add(
+        @Arg("title") title: string,
+        @Arg("time") time: string,
+        @Arg("description") description: string,
+        @Arg("videoChatLink") link: string
+    ) 
+    {
+      try {
+         await Event.insert({
+             title,
+             time,
+             description,
+             videoChatLink:link
+         })
+     } catch (error) {
+         console.log(error);
+         return false;
+     }
+
+    return true
+   }
+
+@Mutation(() => Boolean)
+    async delete(
+        @Arg("id") id: string
+    ) 
+    {
+      try {
+         const event = await Event.findOne(id);
+         if(event) { Event.remove(event);} else {console.log("Event does not exist");}
+      
+     } catch (error) {
+         console.log(error);
+         return false;
+     }
+    return true
+}
+
+@Mutation(() => Boolean)
+    async edit(
+        @Arg("id") id: string
```

Comment:
You should add another argument passed to the edit function that tells what should be updated
From `https://codetain.com/blog/implementing-server-side-crud-with-typescript-typeorm-and-graphql`: 

Here is an example:
![image](https://user-images.githubusercontent.com/52944725/98772218-d1baeb80-23ab-11eb-89e3-2b577716126e.png)
You don't need to assign to the object, you can just pass the data as the second parameter to the Event.update, similar to how you do it now.
The `UpdateEventInfo` should look similar to this except it should have the parameters of the Event that we would want to change.
![image](https://user-images.githubusercontent.com/52944725/98772457-53127e00-23ac-11eb-99e1-f3fe33910c75.png)
Put this where the `NewEvent` class is now in this file.